packages/mobile/src/App.tsx

```diff
@@ -76,12 +70,11 @@ const App = () => {
         <ToastContextProvider>
           <ErrorBoundary>
             <WebRefContextProvider>
-              <WebAppManager webApp={<WebApp webRef={webRef} />}>
+              <WebAppManager webRef={webRef}>
                 <ThemeProvider>
                   <NavigationContainer>
                     <GoogleCast webRef={webRef} />
-                    <AppNavigator />
-                    <SearchScreenLegacy />
```

Comment:
So this means the webview version of the app will no longer work right? I'm wondering if we should maybe merge this into a branch so we don't prevent ourselves from pushing bugfixes the the current app

We ran into that case last week with the fingerprint stuff needing to be released to prod

Reply:
@sliptype so totally agreed, however i was talking with @raymondjacobson whether we should keep things backwards compatible, and he seemed to think no. like all the notification changes that were just merged are not backwards-compat with webview version i beleive

Reply:
Ah ok fair enough, we'll do it live 😤